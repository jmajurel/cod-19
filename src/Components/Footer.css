.contacts {
  padding-top: 0.5rem;
}

.contacts .fab {
  font-size: 1.6rem;
  padding: 0 1.2rem;
  text-decoration: none;
  color: black;
}
.footer {
  position: relative;
  max-width: 100%;
  font-size: 0.7rem;
}
