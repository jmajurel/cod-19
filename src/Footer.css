.contacts {
  padding-top: 0.5rem;
}

.contacts .fab {
  font-size: 1.6rem;
  padding: 0 1.2rem;
  text-decoration: none;
  color: black;
}
.footer {
  position: relative;
  width: 100vw;
  font-size: 0.7rem;
}
