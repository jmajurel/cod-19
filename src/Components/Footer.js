import React from "react";
import "./Footer.css";
const Footer = () => (
  <div className="footer">
    <p>
      Please note this website does not provide any profesional medical advice,
      contact your local medical autority instead
    </p>
    <p>
      This website has been created by JF MAJUREL using public information in
      the aim of fighting COVID-19
    </p>
    <div className="contacts">
      <a
        className="contact"
        href="https://www.linkedin.com/in/jean-f%C3%A9lix-majurel-a142235a/"
      >
        <i className="fab fa-linkedin" />
      </a>
      <a className="contact" href="https://github.com/jmajurel">
        <i className="fab fa-github" />
      </a>
    </div>
  </div>
);

export default Footer;
