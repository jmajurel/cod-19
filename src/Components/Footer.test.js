import React from "react";
import Footer from "./Footer";
import renderer from "react-test-renderer";

test("display", () => {
  const component = renderer.create(<Footer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
