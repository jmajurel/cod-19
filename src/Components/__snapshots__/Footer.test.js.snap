// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display 1`] = `
<div
  className="footer"
>
  <p>
    footer.note
  </p>
  <p>
    footer.author
  </p>
  <p>
    footer.bckImgCredit
  </p>
  <div
    className="contacts"
  >
    <a
      className="contact"
      href="https://www.linkedin.com/in/jean-f%C3%A9lix-majurel-a142235a/"
    >
      <i
        className="fab fa-linkedin"
      />
    </a>
    <a
      className="contact"
      href="https://github.com/jmajurel"
    >
      <i
        className="fab fa-github"
      />
    </a>
  </div>
</div>
`;
